applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_node_utilities.cpp

```diff
@@ -29,12 +29,26 @@ void AddAcceleration(Node& rNode)
     rNode.AddDof(ACCELERATION_Z);
 }
 
+PointerVectorSet<Node, IndexedObject> NodeContainerWithOneFixedDOF(const intrusive_ptr<Node>& pNode)
```

Comment:
I feel this name doesn't reflect which degree of freedom is fixed, whereas the implementation is very clear about that. So we may want to rename this helper function to something like `CreateNodeContainerWithFixedAccelerationY`:

```suggestion
PointerVectorSet<Node, IndexedObject> CreateNodeContainerWithFixedAccelerationY(const intrusive_ptr<Node>& pNode)
```

Reply:
done